src/modules/Message/utils/tokens/__tests__/tokenizeMessage.spec.ts

```diff
@@ -16,7 +16,7 @@ describe('tokenizeMessage', () => {
       messageText: 'Hello world https://example.com',
     });
     expect(tokens).toEqual([
-      { type: 'string', value: 'Hello world' },
+      { type: 'string', value: 'Hello world ' },
```

Comment:
There'll be a trailing white space before a URL comes.

Reply:
Could you leave a comment for this?